### DISCUSSION TITLE
C29 Not changing Probing Temperature

### DISCUSSION THREAD

**Discussion by npaintin** [mriscoc/Ender3V2S1#729] [General]
I issue a C29 L25.00 R195.00 F25.00 B195.00 N5 T50 in the terminal of Octoprint but the probing temperature. (T50) will not change, no matter what I set it to.

**Comment by mriscoc** [comment]
Verified and it is working correctly.

**Comment by npaintin** [comment]
If i use the precompiled Ender3V2-423-BLTUBL-MPC-20221222.bin C29 does work correctly, but if I compile my own using the configuration files from Special_Configurations/Ender3V2-427-BLTUBL-MPC/ (without any changes) it will compile without any errors but the C29 command does not work correctly and will not change the Bed Temperature, but the Grid Points and mesh insert setting work correctly.

**Reply by mriscoc** [reply]
You will receive a response in the terminal confirming the changes.

**Reply by npaintin** [reply]
Sorry all a bit new to this, what do you mean ?

**Reply by mriscoc** [reply]
Octoprint terminal

**Reply by npaintin** [reply]
Got it, yes  I  get Recv: echo:  C29 L25.00 R19.00 F25.00 B195.00 N5 T10 ; 5x5  T=10°C  but when I check the Printer Display under Advanced Settings/Mesh Levelling/Bed Temp, bed temp had not actually changed, all the other setting  will change but not the Bed Temp.
